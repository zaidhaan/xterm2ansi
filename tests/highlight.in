[38;5;162m[m[38;5;246m    1 [m[38;5;162m[m[3;38;5;80m# This is a comment[m
[38;5;246m    2 [m[38;5;162m[m
[38;5;246m    3 [m[38;5;162m[m[1;38;5;43mfunction[m [38;5;162mgreet[m[38;5;184m() {[m
[38;5;246m    4 [m[38;5;162m[m  [1;38;5;28mlocal[m [38;5;162mname[m[38;5;184m=[m[38;5;162m[m[1;38;5;113m$1[m
[38;5;246m    5 [m[38;5;162m[m  [1;38;5;28mecho[m [38;5;162m[m[38;5;179m"Hello,[m [38;5;202m$name[m[38;5;179m!"[m[38;5;162m[m
[38;5;246m    6 [m[38;5;162m[m[38;5;184m}[m
[38;5;246m    7 [m[38;5;162m[m
[38;5;246m    8 [m[38;5;162m[m[1;38;5;43mfor[m [38;5;162mi[m [1;38;5;43min[m [38;5;162m[m[38;5;184m{[m[38;5;162m[m[38;5;32m1[m[38;5;162m.[m[38;5;32m.10[m[38;5;162m[m[38;5;184m};[m [38;5;162m[m[1;38;5;43mdo[m
[38;5;246m    9 [m[38;5;162m[m  [1;38;5;43mif[m [38;5;162m[m[38;5;184m[[m [38;5;162m[m[1;38;5;113m$i[m [38;5;162m[m[1;38;5;28m-eq[m [38;5;162m[m[38;5;32m3[m [38;5;162m[m[38;5;184m] || [[m [38;5;162m[m[1;38;5;113m$i[m [38;5;162m[m[1;38;5;28m-eq[m [38;5;162m[m[38;5;32m7[m [38;5;162m[m[38;5;184m];[m [38;5;162m[m[1;38;5;43mthen[m
[38;5;246m   10 [m[38;5;162m[m    [1;38;5;28mecho[m [38;5;162m[m[38;5;179m"Skipping number[m [38;5;202m$i[m[38;5;179m"[m[38;5;162m[m
[38;5;246m   11 [m[38;5;162m[m    [1;38;5;28mcontinue[m
[38;5;246m   12 [m[38;5;162m[m  [1;38;5;43melif[m [38;5;162m[m[38;5;184m[[m [38;5;162m[m[1;38;5;113m$i[m [38;5;162m[m[1;38;5;28m-gt[m [38;5;162m[m[38;5;32m5[m [38;5;162m[m[38;5;184m];[m [38;5;162m[m[1;38;5;43mthen[m
[38;5;246m   13 [m[38;5;162m[m    [1;38;5;28mecho[m [38;5;162m[m[38;5;179m"Number[m [38;5;202m$i[m [38;5;179mis greater than 5:[m [38;5;202m$(date)[m[38;5;179m"[m[38;5;162m[m
[38;5;246m   14 [m[38;5;162m[m    [1;38;5;28mbreak[m
[38;5;246m   15 [m[38;5;162m[m  [1;38;5;43melse[m
[38;5;246m   16 [m[38;5;162m    greet[m [38;5;179m"user"[m[38;5;162m[m
[38;5;246m   17 [m[38;5;162m[m  [1;38;5;43mfi[m
[38;5;246m   18 [m[38;5;162m[m[1;38;5;43mdone[m
[38;5;246m   19 [m[38;5;162m[m
