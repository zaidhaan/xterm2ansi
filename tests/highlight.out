[35m[0m[90m    1 [0m[35m[0m[3;36m# This is a comment[0m
[90m    2 [0m[35m[0m
[90m    3 [0m[35m[0m[1;36mfunction[0m [35mgreet[0m[33m() {[0m
[90m    4 [0m[35m[0m  [1;32mlocal[0m [35mname[0m[33m=[0m[35m[0m[1;90m$1[0m
[90m    5 [0m[35m[0m  [1;32mecho[0m [35m[0m[33m"Hello,[0m [91m$name[0m[33m!"[0m[35m[0m
[90m    6 [0m[35m[0m[33m}[0m
[90m    7 [0m[35m[0m
[90m    8 [0m[35m[0m[1;36mfor[0m [35mi[0m [1;36min[0m [35m[0m[33m{[0m[35m[0m[36m1[0m[35m.[0m[36m.10[0m[35m[0m[33m};[0m [35m[0m[1;36mdo[0m
[90m    9 [0m[35m[0m  [1;36mif[0m [35m[0m[33m[[0m [35m[0m[1;90m$i[0m [35m[0m[1;32m-eq[0m [35m[0m[36m3[0m [35m[0m[33m] || [[0m [35m[0m[1;90m$i[0m [35m[0m[1;32m-eq[0m [35m[0m[36m7[0m [35m[0m[33m];[0m [35m[0m[1;36mthen[0m
[90m   10 [0m[35m[0m    [1;32mecho[0m [35m[0m[33m"Skipping number[0m [91m$i[0m[33m"[0m[35m[0m
[90m   11 [0m[35m[0m    [1;32mcontinue[0m
[90m   12 [0m[35m[0m  [1;36melif[0m [35m[0m[33m[[0m [35m[0m[1;90m$i[0m [35m[0m[1;32m-gt[0m [35m[0m[36m5[0m [35m[0m[33m];[0m [35m[0m[1;36mthen[0m
[90m   13 [0m[35m[0m    [1;32mecho[0m [35m[0m[33m"Number[0m [91m$i[0m [33mis greater than 5:[0m [91m$(date)[0m[33m"[0m[35m[0m
[90m   14 [0m[35m[0m    [1;32mbreak[0m
[90m   15 [0m[35m[0m  [1;36melse[0m
[90m   16 [0m[35m    greet[0m [33m"user"[0m[35m[0m
[90m   17 [0m[35m[0m  [1;36mfi[0m
[90m   18 [0m[35m[0m[1;36mdone[0m
[90m   19 [0m[35m[0m
